src/components/TotalForcing.jl

```diff
@@ -8,8 +8,8 @@
     exf_excessforcing = Parameter(index=[time], unit="W/m2")
     fs_sulfateforcing = Parameter(index=[time, region], unit="W/m2")
 
-    area = Parameter(index=[region], unit="km2")
-    area_e_eartharea = Parameter(unit="km2", default=5.1e8)
+    area = Parameter(index=[region], unit="km^2")
```

Comment:
@jrising  I'm tandardizing the units of `area` across components so that your current code won't error, you could always use the `ignoreunits` flag in `connect_param!` but since this is done within `set_leftover_params!` this seems like the easiest way to do it.  I also updated all of teh `area_e_eartharea` ones to be `km^2` as well just to keep it clean.

Reply:
That's great.